multitenancy/tenant-manager-datamodel/src/main/resources/META-INF/openapi.json

```diff
@@ -259,22 +262,67 @@
                     "organizationId": {
                         "description": ""
                     },
-                    "authServerUrl": {
-                        "description": "Http endpoint for the auth server (including realm) to be used for this tenant to authenticate against the registry",
-                        "type": "string"
-                    },
-                    "authClientId": {
-                        "description": "ClientId in the authentication server to be used by the registry to authenticate incoming requests made by the tenant",
-                        "type": "string"
+                    "resources": {
+                        "description": "The list of resources that this tenant has available",
+                        "type": "array",
+                        "items": {
+                            "$ref": "#/components/schemas/TenantResource"
+                        }
                     }
                 },
                 "example": {
                     "tenantId": "12345",
                     "createdOn": "2021-03-17T21:35:00Z",
                     "createdBy": "usernamefoo",
-                    "organizationId": "bank-of-foo"
+                    "organizationId": "bank-of-foo",
+                    "resources": [
+                        {
+                            "type": "MAX_TOTAL_SCHEMAS",
+                            "limit": 1000
+                        }
+                    ]
                 },
                 "x-codegen-package": "io.apicurio.multitenant.api.datamodel"
+            },
+            "TenantResource": {
+                "title": "Root Type for TenantResource",
+                "description": "Configuration of the limits for a specific resource type",
+                "required": [
+                    "limit",
+                    "type"
+                ],
+                "type": "object",
+                "properties": {
+                    "type": {
+                        "$ref": "#/components/schemas/ResourceType",
+                        "description": "The type of the resource to limit"
+                    },
+                    "limit": {
+                        "format": "int64",
+                        "description": "The quantity to limit this resource",
+                        "type": "integer"
+                    }
+                },
+                "example": {
+                    "type": "MAX_TOTAL_SCHEMAS",
+                    "limit": 1000
+                }
+            },
+            "ResourceType": {
+                "description": "The list of possible types of a resource that can be limited",
+                "enum": [
+                    "MAX_TOTAL_SCHEMAS",
```

Comment:
It's important for users to know which unit is used for each type, e.g. `*_SIZE_BYTES` is great. Should we make this consistent even for types where the unit can be inferred? I mean:
```
MAX_ARTIFACTS -> MAX_ARTIFACT_COUNT
MAX_NAME_LENGTH -> MAX_NAME_LENGTH_CHARS
```
Also, `MAX_NAME_LENGTH` sounds ambiguous, is it the Artifact name?

Reply:
really good idea